src/main/java/stream/Exercise03.java

```diff
@@ -0,0 +1,23 @@
+package stream;
+
+/**
+ * Exercise03: Write a stream program to read the digits of an integer, the result is a String.
+ * For example, entering 123, the result is "One, Two, Three"
+ */
+public class Exercise03 {
+    private static final String[] digitWords = {
```

Comment:
The variable name here should be in UPPERCASE letters.

Reply:
https://www.oracle.com/docs/tech/java/codeconventions.pdf

Following this source